models/players.js

```diff
@@ -135,6 +144,19 @@ formatEUSoccerPlayers = function(response){
   return players;
 }
 
+formatMLBPlayers = function(response, team_id){
+  parseString(response, function (err, result) {
+    var str = result[Object.keys(result)[0]];
+        for (j=0; j < str.team.length; j++){ 
+          if (str.team[j].$.id == team_id){
```

Comment:
is this just to be extra cautious? If so, maybe add an else clause outputting a message?